src/NzbDrone.Api/Profiles/ProfileSchemaModule.cs

```diff
@@ -27,7 +27,7 @@ private List<ProfileResource> GetAll()
                 .ToList();
 
             var profile = new Profile();
-            profile.Cutoff = Quality.Unknown;
+            profile.Cutoff = Quality.Unknown.Id;
```

Comment:
Yes, it's quality or group ID. We could switch it to the index of the array, but we'd need to keep it updated whenever we re-order qualities so the UI would still need to track it by ID, but we'd store the index.